test/files.test.ts

```diff
@@ -324,27 +347,65 @@ test('upload file if new (list returns no key)', async () => {
   // We'll just have to assume the contents are correct
 });
 
-test('successful run does not need to query account ID', async () => {
```

Comment:
This test is deleted because we actually do need to query account ID now, even for success paths.

Reply:
Its back now.